index.html

```diff
@@ -0,0 +1,102 @@
+            <div class="hamburg">
+                <div class="hamburg1"></div>
+                <div class="hamburg2"></div>
+                <div class="hamburg3"></div>
+            </div>
+            <ul class="menu">
+                <li><a class="menu-item" href="index.html">home</a></li>
+                <li><a class="menu-item" href="#programs">program</a></li>
+                <li><a class="menu-item" href="#ourTeacher">Teachers</a></li>
+                <li><a class="menu-item" href="About.html">About</a></li>
+            </ul>    
+        </nav>
+    </header>
+        <section class="kamuclass">
+            <h1>Are you intersted <br>to Speak many languages?</h1>
+            <h2>Do you want to know many languages?</h2>
+            <p>we teach how to speak fluently each language , like french ,english,spanish,arbic, and more.</p>
+            <p class="stating-date">starting form 10 january 2023</p>
+            <p>we get stated in 2023 </p>
+        </section>
+        <section id="programs" class="programsclass">
+            <h2>programs</h2>
+            <hr class="linebar">
+    <div class="bloc-programs">
+        <div class="bloc1">
+            <img src="img-folder/Icons-program.jpg" alt="" width="100px">
+            <h3>4months class program</h3>
+            <p>Is a program that you can learn with other student everyday, in this program you have to come everyday so that you can be on page with others.</p>
+        </div>
+        <div  class="bloc1">
+            <img src="img-folder/icon-program-wekly.png" alt="" width="100px">
+            <h3>12months class personal program</h3>
+            <p>This program is 12 months because hours are limited as you can work and come to the training center by that you need to add some additional hours to learn.</p>
+        </div>
+        <div  class="bloc1">
+            <img src="img-folder/icon-program6m.jpg" alt="" width="100px">
+            <h3>Night schedule program</h3>
+            <p>in the night program a student must learn during night time he must done also 10months as in night time you are going to learn many houurs than the one for 12 months</p>
+        </div>
+        <div  class="bloc1">
+            <img src="img-folder/icon-programNight.jpg" alt="" width="100px">
+            <h3>weekly home program</h3>
+            <p>In this program the studies is scheduled per week , for them they have to be there once per week means they should be active so that they can lear faster.</p>
+        </div>
+        <div  class="bloc1">
+            <img src="img-folder/icons-program12.png" alt="" width="100px">
+            <h3>Online program</h3>
+            <p>the online program student must have their ouwn computer, and the camra of the computer shoulbe in good quality and question are the most welcome .</p>
+        </div>
+    </div>
+            <h2 class="seeall"><a href="#">see all the programs</a></h2>
+            <hr class="linebar2">
+        </section>
```

Comment:
- Your programs section looks great however, I would suggest that center the items properly in mobile view so they look similar to that of the given design and add some padding around the elements so the spacing around them is similar to that of the given design:+1: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/225416880-17cd05ec-2268-4af7-8062-07c586448128.png)

- The Given Design
![image](https://user-images.githubusercontent.com/102699958/225416950-2226002e-d897-4e7e-8890-a5eb13bc4d49.png)